### PR TITLE
why i picked angular

### PR DESCRIPTION
**Pull request by joelturnbull**

### CODE REVIEW

**Comment by cdmwebs** [comment]
:metal:

File: posts/2013-06-27-why-i-picked-angular.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+I chose [AngularJS](http://angularjs.org/) because it seemed closer to what I know. 
+More like Backbone. [Data-binding](http://docs.angularjs.org/guide/dev_guide.templates.databinding)
+is touted as Angular's killer feature over Backbone, but there's a lot more to
+Angular. I also chose Angular because everybody else here in the shop was
+learning Ember. What is wrong with me?
+
```
Comment (mguterl) [inline posts/2013-06-27-why-i-picked-angular.md:6]: What about adding a subheading here for Templates? I mentioned adding more formatting to our posts to @cdmwebs and @rockwood yesterday and @rockwood seemed to agree.

File: posts/2013-06-27-why-i-picked-angular.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+single source of truth, and eliminating state. It feels like this would be easy
+in Ember.
+
+The Data problem is the arena of Angular [services](http://docs.angularjs.org/guide/dev_guide.services.creating_services). You write Angular services to
+share things between controllers in your app. In my case it was a collection of
+Post models and a collection of Author models. That was my data and every
+controller in my app needed access to it. You hear a lot about dependency
+injection. Services is where that comes into play. When you define a
+controller you also define what services it's going to need, and pass those in.
+
```
Comment (mguterl) [inline posts/2013-06-27-why-i-picked-angular.md:36]: Maybe Documentation as a subheading here?

**Comment by mguterl** [comment]
:thumbsup: :horse: :hamburger:

**Comment by cdmwebs** [comment]
@mguterl is that a horsemeat hamburger?

**Comment by mguterl** [comment]
Yep! :8ball: :sleepy:

**Comment by xero6** [comment]
This is really timely Joel!  I'm scheduling a lunch with Adam from Raco Wireless next week to talk about how we might be able to help them with their Angular project.  I think you should come to this lunch with me and talk more about your experiences.  Adam is very interested in hearing them.

Dewayne Greenwood
Managing Partner

Gaslight
M 513.417.6853
O 513.898.0621
E dewayne@gaslight.co

**Pull request merged**